projects/swimlane/ngx-charts/src/lib/models/chart-data.model.ts

```diff
@@ -16,6 +16,7 @@ export interface SingleSeries extends Array<DataItem> {}
 export interface Series {
   name: StringOrNumberOrDate;
   series: DataItem[];
+  extra?: any;
```

Comment:
is `extra` used elsewhere or just for the `gradientDirection` property?

consider something like:
```suggestion
  extra?: {
    gradientDirection?: string;
  };
```